cypress/e2e/shared/explorer.test.ts

```diff
@@ -450,16 +450,15 @@ describe('DataExplorer', () => {
           .within(() => {
             cy.get('textarea').type('foo |> bar', {force: true})
 
-            cy.get('.squiggly-error').should('be.visible')
+            // TODO the new flux lsp doesn't support squigglies.
+            // Turn this back on after [INSERT ISSUE] is fixed
```

Comment:
@rockstar reminder to both of us to put the link to the squiggily issue here before we merge this